content/docs/reference/authorize-log-fields.mdx

```diff
@@ -54,17 +54,17 @@ The table below lists all available authorize log fields:
 | `email` | The logged-in user's email address | Yes |
 | `headers` | The complete set of HTTP request headers | No |
 | \* `headers.{CustomHeaderName}` | An HTTP Request Header identified by the `HeaderName` | No |
-| `host` | The HTTP Authority. Can be a hostname, domain name, or IP address and may contain a port number (for example, `www.example.com`) | Yes |
+| `host` | The HTTP request `:authority` or `Host` header value. Can be a domain name or IP address and may contain a port number (for example, `www.example.com`) | Yes |
 | `id-token` | The user's ID token | No |
 | `id-token-claims` | The user's ID token claims | No |
 | `impersonate-email` | If [impersonating](/docs/capabilities/impersonation), the impersonated email | Yes |
 | `impersonate-session-id` | If [impersonating](/docs/capabilities/impersonation), the impersonated session ID | Yes |
 | `impersonate-user-id` | If [impersonating](/docs/capabilities/impersonation), the impersonated user ID | Yes |
 | `ip` | The user's IP address | Yes |
```

Comment:
I think what we log for this value may vary depending on the [`xff_num_trusted_hops`](https://www.pomerium.com/docs/reference/the-number-of-trusted-hops) setting, so it might be good to link to that here.